public/pages/CreateMonitor/components/MonitorExpressions/expressions/utils/whereHelpers.js

```diff
@@ -47,12 +47,59 @@ export const displayText = (whereValues) => {
   }
 };
 
-export const validateRange = (value, whereFilters) => {
-  if (value === '') return 'Required';
-  if (whereFilters.fieldRangeEnd < value) {
-    return 'Start should be less than end range';
-  }
-  if (value < whereFilters.fieldRangeStart) {
-    return 'End should be greater than start range';
+export const validateRange = (value = '', filter = FORMIK_INITIAL_WHERE_EXPRESSION_VALUES) => {
+  if (_.isEmpty(value.toString())) return 'Required.';
+  if (filter.fieldRangeEnd < value) return 'Start should be less than end range.';
+  if (value < filter.fieldRangeStart) return 'End should be greater than start range.';
+  return undefined;
+};
+
+export const validateWhereFilter = (filter = FORMIK_INITIAL_WHERE_EXPRESSION_VALUES) => {
+  const fieldName = _.get(
+    filter,
+    'fieldName[0]',
```

Comment:
why only check the first value?

Reply:
The combo box object returns an array of selected objects, even when the `singleSelection` setting for the box is `true`.